layouts/shortcodes/iframe.html

```diff
@@ -0,0 +1,5 @@
+{{ $url := .Get "url" }}
+<div style="width: 100%; height:calc(100vh - 200px)">
+    <iframe src="{{ $url }}" height="100%" width="100%"><a href="{{ $url }}">API Documentation</a></iframe>
```

Comment:
None that I can think of, how about adding a plain link above the iframe incase someone just wants the docs website without the iframe?